data/contributors/organizations.json

```
@@ -580,5 +580,53 @@
         "last_contribution": "2025-11-19"
       }
     }
+  },
+  {
+    "name": "Camptocamp",
+    "subtitle": "Innovative solutions by Open Source experts",
+    "image": "/images/logo-with-text.svg",
+    "width": "350px",
```

Please keep this to 200px as mentioned here: https://qgis.org/community/contributors/add-organisation/#field-guide
```suggestion
    "width": "200px",
```